### ISSUE TITLE
Accessibility Violations sending Issues and not Violation counts to InfluxDB

### ISSUE THREAD

**Issue by 2343909** [sitespeedio/sitespeed.io#4137]
### Your question

Hi
I am trying to set up Accessibility violations in Grafana and based off the violations to send alerts but the counts of the Issues are sent to the InfluxDB and not actual counts of the Violations. The default thresholds are set based off the Violations and not Issues, unless I am wrong here. 
Is there a way to fix this where Violations counts are sent to the InfluxDB or both, issues and violations. 

![image](https://github.com/sitespeedio/sitespeed.io/assets/31040344/7e6b491b-8475-4d1c-9bf0-54e8c72a8469)
![image](https://github.com/sitespeedio/sitespeed.io/assets/31040344/18ecd7a1-329c-4e9b-9c66-2d4b9deeb644)

**Comment by soulgalore** [comment]
Hi @2343909 I think you are right, I think we do not push number of violations to InfluxDB and Graphite. Let me have a look later this week.

**Comment by soulgalore** [comment]
I'm adding a new tag so you can separate them. The InfluxDB names are notperfect, I've been meaning to fix that when I add Prometheus support, but I haven't focused finish that yet, hopefully the coming months.

**Comment by 2343909** [comment]
Great, thanks a lot! 
Yes Prometheus support will be a great addition!
